### ISSUE TITLE
Example on extracting data from a form

### ISSUE THREAD

**Issue by wallclimber21** [escoz/QuickDialog#125]
One of the things that's really missing from the examples is how to interact with the items in a form.

Right now, the example shows how to build the view in a separate Builder object. It also shows how the values of entries can be shown from a block that's called from within that builder. (E.g. UIAlertView example).

But take the following example: Login window with email and password and button.

When email and password are not filled in, the button should be grayed out.

When both are filled in, the button should be activated.

This is clearly all controller material, but there is no example how this should be done in the controller. 

Basically, when the email entry goes out of focus, there should be a trigger to a function to the controller to check the value. etc.

Right now, the example clearly separates the view builder from the controller, but doesn't show how the controller can get back at the data.

Thanks!
